data/mods/gen1/scripts.js

```diff
@@ -530,8 +530,8 @@ let BattleScripts = {
 					if (target.setStatus(moveData.status, pokemon, move)) {
 						target.removeVolatile('mustrecharge');
 						this.hint(
-							'In Gen 1, if a Pokémon that has just used Hyper Beam and has yet to recharge is targeted with a sleep inducing move, ' +
-							'any other status it may already have will be ignored and sleep will be induced regardless.');
+							"In Gen 1, if a Pokémon that has just used Hyper Beam and has yet to recharge is targeted with a sleep inducing move, " +
+							"any other status it may already have will be ignored and sleep will be induced regardless.");
```

Comment:
```suggestion
							"any other status it may already have will be ignored and sleep will be induced regardless."
						);
```

I prefer matching indentation levels for all brackets.

Reply:
I wonder if this is an eslint rule...